test/bridge/testhorton.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+import unittest
+import os, sys
+import numpy
+
+from cclib.bridge import cclib2horton
+from ..test_data import getdatafile
+from cclib.parser.utils import find_package
+
+class HortonTest(unittest.TestCase):
+    """ Tests for the horton bridge in cclib """
+    # Both horton and cclib can read in fchk files. The test routine utilizes this fact and compares the attributes that were directly loaded from each package and the attributes that were converted.
+    
+    def setUp(self):
+        super(HortonTest, self).setUp()
+        
+        self.data, self.logfile = getdatafile("Gaussian", "basicGaussian16", ["dvb_un_sp.fchk"])
+        datadir = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "data"))
+        inputfile = os.path.join(datadir, "Gaussian", "basicGaussian16", "dvb_un_sp.fchk")
+        
+        self._old_horton = False
+        self._found_horton = find_package('horton')
+        self._found_iodata = find_package('iodata')
+        
+        if self._found_horton:
+            try:
+                from horton import __version__
+            except:
+                self._old_horton = True # Old horton versions do not have this __version__ attribute
+            else:
+                if (__version__[0] == '2'):
+                    from horton.io.iodata import IOData
+                    from horton import log
+                    
+                    log.set_level(0) # This suppresses horton outputs so that they do not flood the test log.
+                    self._hortonver = 2
+                    self.iodat = IOData.from_file(inputfile)
+        if self._found_iodata:
+            # Add horton 3 import lines
+            pass
+        
+    def compareArrays(self, arr1, arr2, delta = 1.0e-3):
```

Comment:
I think in other parts of the code we use assertion functions from `numpy.testing`. There should be at least one that is "almost equal" with a delta or tolerance that can be specified.

Reply:
> I think in other parts of the code we use assertion functions from `numpy.testing`. There should be at least one that is "almost equal" with a delta or tolerance that can be specified.

Looks like `numpy.testing.assert_array_almost_equal` would do. Thanks!